spec/jobs/notification/reopen_snoozed_notifications_job_spec.rb

```diff
@@ -13,11 +13,15 @@
   context 'when called' do
     it 'reopens snoozed notifications whose snooze until has passed' do
       described_class.perform_now
+      time_after_job = Time.current
 
       expect(snoozed_till_5_minutes_ago.reload.snoozed_until).to be_nil
       expect(snoozed_till_tomorrow.reload.snoozed_until.to_date).to eq 1.day.from_now.to_date
       expect(snoozed_indefinitely.reload.snoozed_until).to be_nil
+      expect(snoozed_till_5_minutes_ago.reload.last_activity_at).to be_within(1.second).of(time_after_job)
       expect(snoozed_indefinitely.reload.read_at).to be_nil
+      last_snoozed_at = Time.zone.parse(snoozed_till_5_minutes_ago.reload.meta['last_snoozed_at'])
+      expect(last_snoozed_at).to be <= time_after_job
```

Comment:
Expect the value of the snoozed_until field in the earlier step to be equal to the value available in meta.